Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "byteorder"
-version = "0.5.1"  #:version
+version = "0.5.2"  #:version
```

Comment:
Could you remove the version bump please? I like to do that in a separate commit/tag.